config/binutils/binutils.in

```diff
@@ -6,13 +6,17 @@ comment "GNU binutils"
 config BINUTILS_GOLD_SUPPORTS_ARCH
     bool
     default y if ARCH_ARM
+    default y if ARCH_LOONGARCH
+    default y if ARCH_MIPS
+    default y if ARCH_POWERPC
+    default y if ARCH_S390
+    default y if ARCH_SPARC
     default y if ARCH_X86
 
 config BINUTILS_GOLD_SUPPORT
     bool
     default y
     depends on BINUTILS_GOLD_SUPPORTS_ARCH
-    depends on ! STATIC_TOOLCHAIN
```

Comment:
This is my only question about this change. Are you sure that this won't break backwards comparability?

Reply:
I could totally be off point on this.

Reply:
It should be OK. ld.bfd is still the default linker and we've been supporting both ld.bfd and ld.gold for dynamic toolchains so I don't anticipate any problems.

The think I was less sure about was is there a better way of actually building both gold and bfd statically. This seems to be the only way. Various incantations of LDFLAGS would make one work but not the other.